lib/match/runner.rb

```diff
@@ -90,5 +96,20 @@ def profile(params: nil, certificate_id: nil)
 
       return uuid
     end
+
+    def device_count_different?(params: nil, profile: nil)
+      if profile && params[:force_for_new_devices]
+        parsed = FastlaneCore::ProvisioningProfile.parse(profile)
+        uuid = parsed["UUID"]
+
+        portal_profiles = Spaceship.provisioning_profile
+                            .ad_hoc.find_by_bundle_id(params[:app_identifier])
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align `.ad_hoc` with `.provisioning_profile` on line 105.